db/builder.go

```diff
@@ -0,0 +1,29 @@
+package db
+
+import "github.com/pkg/errors"
+
+var (
+	// ErrDBEmptyTrieDBPath is the error when trie db path is empty
+	ErrDBEmptyTrieDBPath = errors.New("Invalid empty trie db path")
```

Comment:
`ErrEmptyDBPath` = errors.New("Invalid empty db path")
this applies to all empty path error, not necessarily/specifically for trie